Sources/Hummingbird/Router/RouterResponder.swift

```diff
@@ -57,3 +57,14 @@ public struct RouterResponder<Context: BaseRequestContext>: HTTPResponder {
         return try await responder.respond(to: request, context: context)
     }
 }
+
+extension RouterResponder: RouterMiddleware {
```

Comment:
I'm not sure I understand the purpose of this change yet. I'm struggling to find any real-world examples that need this.